src/shared/icons/aws.tsx

```diff
@@ -1,7 +1,16 @@
 import Image from 'next/image';
 
 import aws from '@/shared/assets/icons/aws-gray.svg';
+import { LinkCustom } from '@/shared/ui/link-custom';
 
 export const AwsLogo = () => {
-  return <Image src={aws} alt="AWS icon" />;
+  return (
+    <LinkCustom
+      href="https://aws.amazon.com/"
+      external
+      icon={null}
+    >
```

Comment:
I'd wrap the logos in links inside the `Partnered` component so that the icons can be reused without the links if needed